api/v1alpha1/wordpressinstance_types.go

```diff
@@ -37,18 +50,23 @@ type WordpressInstanceSpec struct {
 
 	// Image will be used as image of the container that Wordpress runs in.
 	// If not specified, the default will be used.
+	// +optional
```

Comment:
Is this used when generating the json schema in the CRD?

Reply:
The answer to this is... sort of. If we actually removed the `// +optional` tag here, the CRD generation would not change because the field is also designated as `omitempty`. If we removed `omitempty` and removed the `// +optional` tag the field would be designated as required. If we removed `omitempty` and kept the `// +optional` tag there would be no change. I included it here because it is part of our [best practices](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#pointer-types-and-markers) to do so. However, it is also part of our best practices for this field to be a pointer, but it doesn't really have to be here and I was trying to do minimal possible amount of changes to get this merged :+1: